frontend/package.json

```diff
@@ -13,17 +13,17 @@
   "devDependencies": {
     "@11ty/eleventy": "^0.12.1",
     "express": "^4.17.1",
+    "js-yaml": "^4.0.0",
     "jshint": "^2.12.0",
     "jsonwebtoken": "^8.5.1",
     "morgan": "^1.10.0",
     "rimraf": "^3.0.2"
   },
   "scripts": {
-    "clean": "rimraf ./_site ./dist",
-    "start": "npm -s run serve",
```

Comment:
Am testing this locally and am unable to run the local server without `run serve` on this line or just `serve` further down on line 27. `watch` only looks for changes in the build at :7071 but isn't also running the site.

Is there another way I should be running the site locally?

Reply:
Sorry @drew-usds forgot that you need the serve portion - will add it back in.